src/pages/components/footer.mdx

```diff
@@ -24,176 +26,215 @@ import ResourceLinks from 'components/ResourceLinks';
 
 ## Overview
 
-Default footer includes all high-level corporate navigation, the required legal links, the optional adjunct legal links, the locale selector and branding. This footer is generated automatically, based on the country and language code that you set on your page in the metadata.
+The Footer is a required component on all IBM.com pages. There are three sizes available: [Default](#default), [Short](#short) and [Micro](#micro), and each size has a specific use case. The Footer includes the IBM logo, navigational links, Locale or Language selector and required legal links.
+
+The Footer is generated automatically based on the country and language code that is set on the page in the metadata.
 
 ![Image of default footer](../../images/component/footer/footerV2-default-01.jpg)
 
 <Caption>
   Default footer at 1584px max breakpoint, without optional adjunct legal links
 </Caption>
 
+## Default
+
+The Default footer includes all high-level corporate navigation, required legal links, locale selector, branding, and optional adjunct legal links.
+
 #### When to use
 
 Default footer is suitable for most of the IBM.com pages. Examples include the IBM.com US home page.
 
-## Variations
+### Anatomy
 
-### Short
-
-Short footer removes all the high-level corporate navigation, leaving the required legal links, the optional adjunct legal links, the locale selector and branding.
+![Image of minimal footer](../../images/component/footer/footerV2-default-anatomy.jpg)
 
-![Image of minimal footer](../../images/component/footer/footerV2-short-01.jpg)
+<Caption>
+  Default footer at the 1584px (max) breakpoint with one adjunct legal link
+</Caption>
 
-<Caption>Short footer at 1312px xlg breakpoint</Caption>
+1. **IBM logo:** Links to the IBM homepage and is a required branding element of the Footer. The IBM logo should not be altered.
+1. **Link group heading:** The link group heading is present for all the link groups.
+1. **Link:** The Default footer links are carefuly considered from the main navigation level and should not be changed.
```

Comment:
```suggestion
1. **Link:** The default footer links are carefully considered from the main navigation level and should not be changed.
```

Reply:
will fix carefully in another commit, leaving Default capitalized